vmm/sandbox/config_qemu_x86_64.toml

```diff
@@ -0,0 +1,41 @@
+[sandbox]
+log_level = "info"
+
+[hypervisor]
+machine_accelerators = ""
+firmware_path = ""
+cpu_features = "pmu=off"
+cpu_model = "host"
+qemu_path = "/usr/bin/qemu-system-x86_64"
+machine_type = "virt"
```

Comment:
please check this machine type,  I believe it should be 'pc' instread

Reply:
fixed